### PR TITLE
Text review and update

### LINKED ISSUE #277: QeApp GUI text improvement feedback from Nicola

### PR DESCRIPTION
**Pull request by unkcpz**
fixes #277 

Update text on feedback from Nicola.

### CODE REVIEW

File: aiidalab_qe/process.py [hunk 1]
```diff
@@ -28,7 +28,9 @@ class WorkChainSelector(ipw.HBox):
     FMT_WORKCHAIN = "{wc.pk:6}{wc.ctime:>10}\t{wc.state:<16}\t{wc.formula} \t {wc.relax_info} \t {wc.properties_info}"
 
     def __init__(self, **kwargs):
-        self.work_chains_prompt = ipw.HTML("<b>Select workflow or start new:</b>&nbsp;")
+        self.work_chains_prompt = ipw.HTML(
+            "<b>Select computed workflow or start a new:</b>&nbsp;"
```
Comment (mbercx) [inline aiidalab_qe/process.py:32]: I'd still add "one" here at the end ("start a new one"). But the sentence is already taking up quite a bit of space as it is 😅

File: aiidalab_qe/static/welcome.jinja [hunk 2]
```diff
@@ -9,10 +9,10 @@
 <div style="padding-top: 0px; padding-bottom: 0px; line-height: 140%;">
   <h3>Welcome to the AiiDAlab Quantum ESPRESSO app! 👋</h3>
 
-  The <a href="https://www.quantum-espresso.org/" target="_blank">Quantum ESPRESSO</a> app (or QE app for short) is a graphical interface for calculating material properties based on density-functional theory (DFT).
-  Each of the properties is calculated by workflows powered by the <a href="https://www.aiida.net/" target="_bland"> AiiDA engine</a>, and maintained in the <a href="https://aiida-quantumespresso.readthedocs.io/en/latest/" target="_blank"> Quantum ESPRESSO plugin</a> for AiiDA.
+  The <a href="https://www.quantum-espresso.org/" target="_blank">Quantum ESPRESSO</a> app (or QE app for short) is a graphical front end for calculating materials properties using Quantum ESPRESSO (QE).
+  Each of the property is calculated by workflows powered by the <a href="https://www.aiida.net/" target="_bland"> AiiDA engine</a>, and maintained in the <a href="https://aiida-quantumespresso.readthedocs.io/en/latest/" target="_blank"> Quantum ESPRESSO plugin</a> for AiiDA.
```
Comment (mbercx) [inline aiidalab_qe/static/welcome.jinja:13]: This change seems incorrect? Maybe "Each property"?

File: aiidalab_qe/steps.py [hunk 3]
```diff
@@ -48,13 +48,17 @@ class WorkChainSettings(ipw.VBox):
...
+        (2) Atomic positions: perform a full relaxation of the internal atomic coordinates. <br>
+        (3) Full geometry: perform a full relaxation for both the internal atomic coordinates and the cell vectors. </div>"""
     )
     materials_help = ipw.HTML(
         """<div style="line-height: 140%; padding-top: 10px; padding-bottom: 10px">
-        Below you can indicate both if the material is magnetic and a metal or insulator. For now only ferromagnetic configurations are possible, since antiferromagnetism is more complicated to study automatically. If you're not sure whether your material is insulating, choose "Metal", since the corresponding settings usually also work quite well for insulators.
+        Below you can indicate both if the material should be treated as an insulator
+        or a metal (if in doubt, choose "Metal"),
+        and if it should be studied with magnetization/spin polarization,
+        switched magnetism ON or OFF (ON is at least twice more costly).
```
Comment (mbercx) [inline aiidalab_qe/steps.py:61]:
```suggestion
        switch magnetism ON or OFF (ON is at least twice more costly).
```

File: aiidalab_qe/steps.py [hunk 4]
```diff
@@ -165,7 +168,8 @@ class SmearingSettings(ipw.VBox):
     smearing_description = ipw.HTML(
         """<p>
         The smearing type and width is set by the chosen <b>protocol</b>.
-        Tick the box to override the default.
+        Tick the box to override the default, not advised unless you master <b>smearing effects</b> (click <a href="http://theossrv1.epfl.ch/Main/ElectronicTemperature"
```
Comment (mbercx) [inline aiidalab_qe/steps.py:171]: Perhaps
```suggestion
        Tick the box to override the default, not advised unless you've mastered <b>smearing effects</b> (click <a href="http://theossrv1.epfl.ch/Main/ElectronicTemperature"
```

**Review by mbercx — CHANGES_REQUESTED** [review]
Thanks @unkcpz! I've left a few comments.

File: aiidalab_qe/process.py (hunk 1 above)
Comment (unkcpz) [reply aiidalab_qe/process.py:32]: There is a lot of space for this line, but I still prefer without 'one' which is more straightforward.

File: aiidalab_qe/steps.py (hunk 4 above)
Comment (unkcpz) [reply aiidalab_qe/steps.py:171]: I agree!

**Review by unkcpz — COMMENTED** [review]
@mbercx thanks! I am for sure not able to judge which phrase is better because of my poor ENG writing 😆 But I think we can loose a bit, I accept some requests and it is good to go I guess.

File: aiidalab_qe/static/welcome.jinja (hunk 2 above)
Comment (unkcpz) [reply aiidalab_qe/static/welcome.jinja]: Changed

**Review by mbercx — APPROVED** [review]
All good! 🚀

File: aiidalab_qe/process.py (hunk 1 above)
Comment (csadorf) [reply aiidalab_qe/process.py:32]: But this is grammatically incorrect?

File: aiidalab_qe/pseudos.py [hunk 5]
```diff
@@ -43,8 +43,7 @@ class PseudoFamilySelector(ipw.VBox):
     )
     dft_functional_help = ipw.HTML(
         """<div style="line-height: 140%; padding-top: 10px; padding-bottom: 10px; opacity:0.5;">
-        The exchange-correlation energy is calculated based on the charge
-        density using this functional. We currently provide support for two
+        The exchange-correlation energy is calculated functional. We currently provide support for two
```
Comment (csadorf) [inline aiidalab_qe/pseudos.py:46]: > The exchange-correlation energy is calculated functional.

That sounds weird to me. Should it say "is calculated via a functional" ?

File: aiidalab_qe/steps.py [hunk 6]
```diff
@@ -76,10 +80,9 @@ class WorkChainSettings(ipw.VBox):
     )
     protocol_help = ipw.HTML(
         """<div style="line-height: 140%; padding-top: 6px; padding-bottom: 0px">
-        The "moderate" protocol represents a balanced trade-off between
+        The "moderate" protocol represents a trade-off between
         accuracy and speed. Choose the "fast" protocol for a faster calculation
-        with less precision and the "precise" protocol that provides more
-        accuracy but will take longer.</div>"""
+        with less precision and the "precise" protocol to aim at best accuracy (at the prece of longer/costlier calculations).</div>"""
```
Comment (csadorf) [inline aiidalab_qe/steps.py:85]: typo: "prece"

File: aiidalab_qe/steps.py [hunk 7]
```diff
@@ -165,7 +168,8 @@ class SmearingSettings(ipw.VBox):
     smearing_description = ipw.HTML(
         """<p>
         The smearing type and width is set by the chosen <b>protocol</b>.
-        Tick the box to override the default.
+        Tick the box to override the default, not advised unless you've mastered <b>smearing effects</b> (click <a href="http://theossrv1.epfl.ch/Main/ElectronicTemperature"
```
Comment (csadorf) [inline aiidalab_qe/steps.py:171]: Why is "HERE" block-capitalized?

**Comment by unkcpz** [comment]
Thanks @csadorf, they are addressed by https://github.com/aiidalab/aiidalab-qe/pull/296

**Pull request merged**